auditwheel/elfutils.py

```diff
@@ -137,3 +137,27 @@ def is_subdir(path: str, directory: str) -> bool:
     if relative.startswith(os.pardir):
         return False
     return True
+
+
+def filter_undefined_symbols(
+    path: str, symbols: Dict[str, List[str]]
+) -> Dict[str, List[str]]:
+    if not symbols:
+        return {}
+    undef_symbols = set("*")
+    with open(path, "rb") as f:
+        elf = ELFFile(f)
+        section = elf.get_section_by_name(".dynsym")
+        if section is not None:
+            # look for all undef symbols
+            for sym in section.iter_symbols():
+                if sym["st_shndx"] == "SHN_UNDEF":
+                    undef_symbols.add(sym.name)
```

Comment:
nit: This could be made into a utility function to find undefined symbols. That would make this function more readable as well.

Reply:
will do.

Reply:
done